### ISSUE TITLE
Bot messages logging

### ISSUE THREAD

**Issue by aicynide** [Charibdys/Private-Parlor#99]
A channel to automate bot message archive in the similar way desuarchive,archiveofsin,4pleb does

**Comment by Charibdys** [comment]
This does not seem applicable to Private Parlor. One of the key features of the anonymous chat bot is that it is amnesiac. It does not permanently store a detailed record of user messages, aside from recording data about who sent a message, which message it is, who received it, who liked it, and who warned it. When the bot restarts, it knows nothing about the messages it relayed (unless it's using the database to store message history, which records equally limited information about messages and deletes their records in time).

As far as implementation goes, the user could not get all of the messages at once. This would quickly produce a rate limit as there could be 10s of messages or more within an interval of time. Any messages the user does get would have to be mixed in with new/incoming messages, making it hard to follow which messages are the ones from the archive. 

Some of those problems can be resolved, but in any case, the usecase for this feature is very limited. Users expect messages sent before they've started the bot to be unavailable to them. Rarely do users join a chat and ask about messages sent prior to their joining. If they do, someone else typically forwards the message(s) in question to the bot for everyone to see. 

Because of the rare usecase and the above issues, I do not intend to implement this feature.

**Comment by aicynide** [comment]
http://desuarchive.org/
https://4plebs.org/
https://archiveofsins.com

**Comment by Charibdys** [comment]
Archives are useful for imageboards, but they're not particularly useful nor feasible for a Telegram bot.
